saleor/graphql/core/types/common.py

```diff
@@ -59,8 +61,13 @@ class LanguageDisplay(graphene.ObjectType):
     language = graphene.String(description='Language.', required=True)
 
 
+PermissionEnum = graphene.Enum(
+    'PermissionEnum', [(str_to_enum(codename), codename)
```

Comment:
I think we should split it by the dot here, as we did in `split_permission_codename` function, so the final permission will be
`MANAGE_USERS` 
instead of 
`ACCOUNT_MANAGE_USERS`

Reply:
I'll resolve that.